app/src/main/java/com/gacode/relaunchx/ReLaunch.java

```diff
@@ -1683,53 +1642,24 @@ public boolean onSingleTapConfirmed(MotionEvent e) {
 							200, 100, 0);
 					gv.dispatchTouchEvent(ev);
 				} else { // other devices
-					int first = gv.getFirstVisiblePosition();
-					int total = itemsArray.size();
-					int last = gv.getLastVisiblePosition();
-					if (total == last + 1)
-						return true;
-					int target = last + 1;
-					if (target > (total - 1))
-						target = total - 1;
-					RepeatedDownScroll ds = new RepeatedDownScroll();
-					ds.doIt(first, target, 0);
+					if (!prefs.getBoolean("doNotHyph", true)) {
+						setItemsHeights(gv);
```

Comment:
Just remark - it will not work when ScrollJump is used. If you jump to some location than press PgUp the sizes of the items immediately above were not set.

Reply:
I know it, and that's another reason why I found scroll jump functionality not very helpful. The same situation happens when long press: jumps to the very end. But then pgup scrolls to beggining (or at least it should).